### PR TITLE
Joomla 4.0-rc2 & Joomla 3.10-alpha7

### PR DESCRIPTION
**Pull request by zero-24**
- [x] Hashes 3.10-alpha7
- [x] Hashes 4.0-rc2 (@wilsonge)
- [x] Release article ID (@softforge)

### CODE REVIEW

**Comment by softforge** [comment]
The id of the article is 5838
so the url when live will be

https://www.joomla.org/announcements/release-news/5838-joomla-4-rc-2-and-joomla-3-10-alpha-7.html

File: www/core/test/310to4_extension.xml [hunk 1]
```diff
@@ -5,17 +5,17 @@
 		<description>Joomla! 4.0 CMS</description>
 		<element>joomla</element>
 		<type>file</type>
-		<version>4.0.0-rc1</version>
-		<infourl title="Joomla 4.0.0-rc1 Release">https://www.joomla.org/announcements/release-news/5837-joomla-4-0-0-rc1-and-joomla-3-10-alpha6.html</infourl>
+		<version>4.0.0-rc2</version>
+		<infourl title="Joomla 4.0.0-rc2 Release">https://www.joomla.org/announcements/release-news/XXXX-joomla-4-0-0-rc2-and-joomla-3-10-alpha7.html</infourl>
```
Comment (zero-24) [inline www/core/test/310to4_extension.xml:9]:
```suggestion
		<infourl title="Joomla 4.0.0-rc2 Release">https://www.joomla.org/announcements/release-news/5838-joomla-4-0-0-rc2-and-joomla-3-10-alpha7.html</infourl>
```

File: www/core/test/extension_test.xml [hunk 2]
```diff
@@ -44,17 +44,17 @@
 		<description>Joomla! 3.10 CMS</description>
 		<element>joomla</element>
 		<type>file</type>
-		<version>3.10.0-alpha6</version>
-		<infourl title="Joomla 3.10.0-alpha6 Release">https://www.joomla.org/announcements/release-news/5837-joomla-4-0-0-rc1-and-joomla-3-10-alpha6.html</infourl>
+		<version>3.10.0-alpha7</version>
+		<infourl title="Joomla 3.10.0-alpha7 Release">https://www.joomla.org/announcements/release-news/XXXX-joomla-4-0-0-rc2-and-joomla-3-10-alpha7.html</infourl>
```
Comment (zero-24) [inline www/core/test/extension_test.xml:48]:
```suggestion
		<infourl title="Joomla 3.10.0-alpha7 Release">https://www.joomla.org/announcements/release-news/5838-joomla-4-0-0-rc2-and-joomla-3-10-alpha7.html</infourl>
```

File: www/core/test/extension_test.xml [hunk 3]
```diff
@@ -65,17 +65,17 @@
 		<description>Joomla! 4.0 CMS</description>
 		<element>joomla</element>
 		<type>file</type>
-		<version>4.0.0-rc1</version>
-		<infourl title="Joomla 4.0.0-rc1 Release">https://www.joomla.org/announcements/release-news/5837-joomla-4-0-0-rc1-and-joomla-3-10-alpha6.html</infourl>
+		<version>4.0.0-rc2</version>
+		<infourl title="Joomla 4.0.0-rc1 Release">https://www.joomla.org/announcements/release-news/XXXX-joomla-4-0-0-rc2-and-joomla-3-10-alpha7.html</infourl>
```
Comment (zero-24) [inline www/core/test/extension_test.xml:69]:
```suggestion
		<infourl title="Joomla 4.0.0-rc1 Release">https://www.joomla.org/announcements/release-news/5838-joomla-4-0-0-rc2-and-joomla-3-10-alpha7.html</infourl>
```

**Comment by zero-24** [comment]
Applied thanks @softforge

**Pull request merged**